build/gen_stub.php

```diff
@@ -65,6 +65,10 @@ public function __construct(string $name, bool $isBuiltin) {
 
     public static function fromNode(Node $node) {
         if ($node instanceof Node\Name) {
+            if ($node->toString() === "mixed") {
```

Comment:
Workaround until php-ast gets support for `mixed`. I can probably file a PR for that.

Reply:
This needs to be in https://github.com/nikic/PHP-Parser rather than php-ast. But for now the workaround is fine.